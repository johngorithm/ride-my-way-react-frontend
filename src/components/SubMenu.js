import React, { Fragment } from 'react';
import { Link } from 'react-router-dom';


class SubMenu extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <Fragment>
        <div className="submenu row">
          {/**<!--SUBMENU BAR STARTS-->**/}
          <Link
            to="/home"
            className="link active co-xl-6 co-lg-6 co-md-6 co-sm-6 co-xs-6 center-text border"
          >
            RIDE OFFERS
        </Link>
          <Link
            to="/requests"
            className="co-xl-6 co-lg-6 co-md-6 co-sm-6 co-xs-6 center-text"
          >
            MY REQUESTS
        </Link>
        </div>
      </Fragment>
    );
  }
}



export default SubMenu;
