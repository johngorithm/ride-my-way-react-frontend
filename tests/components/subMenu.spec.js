import React from 'react';
import { shallow } from 'enzyme';

import SubMenu from 'components/SubMenu';


describe('SUBMENU COMPONENT TESTS', () => {
  const wrapper = shallow(<SubMenu />);

  test('renders without crashing', () => {
    expect(wrapper.find('Link')).toHaveLength(2);
  })

  test('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
