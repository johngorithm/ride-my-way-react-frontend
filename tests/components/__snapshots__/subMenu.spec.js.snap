// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SUBMENU COMPONENT TESTS matches snapshot 1`] = `
<Fragment>
  <div
    className="submenu row"
  >
    <Link
      className="link active co-xl-6 co-lg-6 co-md-6 co-sm-6 co-xs-6 center-text border"
      replace={false}
      to="/home"
    >
      RIDE OFFERS
    </Link>
    <Link
      className="co-xl-6 co-lg-6 co-md-6 co-sm-6 co-xs-6 center-text"
      replace={false}
      to="/requests"
    >
      MY REQUESTS
    </Link>
  </div>
</Fragment>
`;
